pkg/api/alertmanager.go

```diff
@@ -106,7 +106,7 @@ type GetDeleteSilenceParams struct {
 // swagger:parameters RouteGetSilences
 type GetSilencesParams struct {
 	// in:query
-	Filter []string
+	Filter []string `json:"filter"`
```

Comment:
Does this mean that the query parameter will become `filter` instead of `Filter` in [this](https://github.com/grafana/grafana/pull/32174#discussion_r605133103) request, right?